pages/index.jsx

```diff
@@ -104,6 +109,15 @@ export default function Home({ initialItems }) {
               hidden: true,
               component: () => <MLQ3 item={item} />,
               info: <MLQ3Info />,
+              stats: (
+                <GenderStatsTOT
+                  field="mlq3"
+                  options={[
+                    'Precise algorithm',
+                    'Localization-friendly algorithm',
+                  ]}
+                />
+              ),
```

Comment:
could export this component from the mlq3 file that way you don't have to redefine the options here, alternatively, you could export just the options since you already pull them into their own variable in the mlq3 file